test/lib/rule-finder.js

```diff
@@ -4,30 +4,34 @@ const proxyquire = require('proxyquire');
 
 let ModuleResolver;
 try {
-  // eslint 7.12+
-  ModuleResolver = require('@eslint/eslintrc/lib/shared/relative-module-resolver');
+  // ESLint 7.12+
+  ModuleResolver = require('@eslint/eslintrc').Legacy.ModuleResolver;
```

Comment:
> If there's no deep import way to get to it, then we'll have to mock out the entire eslintrc module.

@ljharb Since`@eslint/eslintrc` is now ESM, it's not possible to deep import anymore and we need to use the publicly available path to get to it.

Can you point me in the direction of how to mock it out?
Is it just a new entry in `getRuleFinder` & `getRuleFinderForDedupeTests`?